content/Software and Downloads/Compilation/The-IDE-prints-a-message-about-sketch-size-and-memory-usage.md

```diff
@@ -2,13 +2,10 @@
 title: "The IDE prints a message about sketch size and memory usage"
 ---
 
-After compiling a sketch the IDE will check for storage and memory usage, against the specifications of the selected board. **Note that the results are always printed to the console, even if successful.** As long as the use of storage space and dynamic memory usage are both below 100%, the message does not indicate a problem.
+The IDE always checks for storage space and dynamic memory usage when compiling your code and prints the results to the console.
 
-## Example
+![IDE storage space and memory usage printed in the console](img/IDE_storage_and_memory_console.png)
 
-```
-Sketch uses 4316 bytes (15%) of program storage space. Maximum is 28672 bytes.
-Global variables use 153 bytes (5%) of dynamic memory, leaving 2407 bytes for local variables. Maximum is 2560 bytes.
-```
+* If both values are **below** 100%, you don't need to do anything.
```

Comment:
We need this in text somewhere in the article, so it shows for users search for the message text.